spec/sprint_teams_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+require 'yaml'
+
+RSpec.describe '_data/sprint_teams.yml' do
+  data = YAML.load_file('_data/sprint_teams.yml')
+  data.each do |team|
+    describe team['name'] do
+      it 'includes required properties' do
```

Comment:
I'd add this or wrap in an `aggregate_failures do ... end` block
```suggestion
      it 'includes required properties', :aggregate_failures do
```